src/wobjectimpl.h

```diff
@@ -35,10 +35,10 @@ template<std::size_t... I1, std::size_t... I2> struct concatenate_helper<std::in
     static constexpr int size = sizeof...(I1) + sizeof...(I2);
     static constexpr auto concatenate(const StaticString<sizeof...(I1)> &s1, const StaticString<sizeof...(I2)> &s2) {
         StaticStringArray<size> d = { s1[I1]... , s2[I2]... };
-        return StaticString<size>( d );
+        return makeStaticString(d);
     }
 };
-constexpr StaticString<1> concatenate(StaticStringList<>) { return {""}; }
+constexpr StaticString<1> concatenate(StaticStringList<>) { return {0}; }
```

Comment:
what's the reason behind this change? Why can't it stay `""` ?
Or maybe `'\0'`  ?

Reply:
`""` is of type `char[1]` that decays to `char*`. This would not allow an initialization.
`\0` would work and also shows more the intend.